extensions/linear/src/unread-notifications.tsx

```diff
@@ -84,7 +88,13 @@ function UnreadNotifications() {
               title={baseTitle}
               subtitle={notification.issue?.title ? truncate(notification.issue.title, 20) : ""}
               tooltip={`${notification.issue?.identifier}: ${notification.issue?.title}`}
-              onAction={() => openNotification(notification)}
+              onAction={async (event: MenuBarExtra.ActionEvent) => {
+                if (event.type === "left-click") {
+                  openNotification(notification);
```

Comment:
Does it need an await here?

```suggestion
                  await openNotification(notification);
```

Reply:
I'm not sure, but it didn't work without for me 😕